libnmstate/nm/applier.py

```diff
@@ -424,7 +424,8 @@ def _build_connection_profile(iface_desired_state, base_con_profile=None):
             )
             settings.append(linux_bridge_setting)
     elif iface_type == ovs.BRIDGE_TYPE:
-        ovs_bridge_options = ovs.translate_bridge_options(iface_desired_state)
```

Comment:
Please explain the reasons for the change in the commit message, see other comment.